python/lsst/ts/observatory/control/maintel/mtcalsys.py

```diff
@@ -339,6 +340,29 @@ async def setup_laser(
         await self.change_laser_optical_configuration(optical_configuration)
         await self.change_laser_wavelength(wavelength, use_projector)
 
+    async def get_projector_setup(self) -> tuple:
```

Comment:
When you do a type hint for a tuple with a known number of values and types, you should add that to the type annotation, e.g.:

```
async def get_projector_setup(self) -> tuple[str, float, float, float]:
```